src/Card.tsx

```diff
@@ -206,28 +234,79 @@ export default function Card({
+        location: draftLocation,
+        website,
+        frequency: draftFrequency != "" ? parseFloat(draftFrequency) : undefined,
+        offset: draftOffset != "" ? parseFloat(draftOffset) : undefined,
+        tone: draftTone != "" ? parseFloat(draftTone) : undefined,
+        rxtone: draftRxTone != "" ? parseFloat(draftRxTone) : undefined,
+      });
+    }
+
+  }, [draftCardType, draftCallsign, draftName, draftWebsite, draftStar, draftLocation, draftFrequency, draftOffset, draftTone, draftRxTone, onEdit]);
 
   const handleCreate = useCallback(() => {
-    onEdit({
-      callsign: draftCallsign,
-      name: draftName,
-      website: draftWebsite || undefined,
-      star: draftStar,
-    });
+    if (draftCardType == "person") {
+      onEdit({
+        cardType: "person",
+        star: draftStar,
+        callsign: draftCallsign,
+        name: draftName,
+        location: draftLocation,
+        website: draftWebsite || undefined,    
+      });
+    }
+    else {
+      onEdit({
+        cardType: "repeater",
+        star: draftStar,
+        callsign: draftCallsign,
+        name: draftName,
+        location: draftLocation,
+        website: draftWebsite || undefined,
+        frequency: draftFrequency != "" ? parseFloat(draftFrequency) : undefined,
+        offset: draftOffset != "" ? parseFloat(draftOffset) : undefined,
+        tone: draftTone != "" ? parseFloat(draftTone) : undefined,
+        rxtone: draftRxTone != "" ? parseFloat(draftRxTone) : undefined,
+      });
+    }
+
+    setDraftCardType("person");
     setDraftCallsign("");
     setDraftName("");
     setDraftWebsite("");
-  }, [draftCallsign, draftName, draftWebsite, draftStar, onEdit]);
+    setDraftLocation("");
+    setDraftFrequency("");
+    setDraftOffset("");
+    setDraftTone("");
+    setDraftRxTone("");
+
+  }, [draftCardType, draftCallsign, draftName, draftWebsite, draftStar, draftLocation, draftFrequency, draftOffset, draftTone, draftRxTone, onEdit]);
 
   const displayCallsign = editMode ? draftCallsign : contact.callsign;
   const displayName = editMode ? draftName : contact.name;
+  const displayLocation = editMode ? draftLocation : contact.location;
+  
+  const displayFreq     = editMode ? draftFrequency : (!Number.isNaN(contact.frequency) && contact.frequency !== undefined) ? contact.frequency.toFixed(FREQ_TRAILING).toString() + " MHz" : "";
+  const displayOffset   = editMode ? draftOffset    : (!Number.isNaN(contact.offset)    && contact.offset    !== undefined) ? (Math.sign(contact.offset) == 1 ? "+" : "") + contact.offset.toFixed(2).toString() + " MHz" : "";
+  const displayToneUp   = editMode ? draftTone      : (!Number.isNaN(contact.tone)    && contact.tone    !== undefined) ? contact.tone.toFixed(TONE_TRAILING).toString()   + " Hz" : "";
+  const displayToneDown = editMode ? draftRxTone    : (!Number.isNaN(contact.rxtone)  && contact.rxtone  !== undefined) ? contact.rxtone.toFixed(TONE_TRAILING).toString() + " Hz" : "";
```

Comment:
autoformat please (https://prettier.io/, or the vscode extension)

Reply:
![image](https://github.com/user-attachments/assets/36a97cd4-3977-4404-82fa-7fd944f1005e)